### ISSUE TITLE
rename package and put it on the CRAN

### ISSUE THREAD

**Issue by timflutre** [Gregor-Mendel-Institute/mlmm#11]
This "mlmm" package now has a synonym on the CRAN, as well as its dependency "emma". It would hence be great to fix this by:

1. renaming this "mlmm" package, for instance as "MultLocMixMod" to avoid any ambiguity (but feel free to propose any other name) -> you can copy the code into another repo, empty this one, keeping only the README indicating the URL of the new repo;

2. including the code from "emma" directly into this package (possible under the LGPL);

3. eventually, putting it on the CRAN (http://r-pkgs.had.co.nz/release.html) would also be great.

**Comment by timeu** [comment]
Pinging @visegura what do you think ? 
I can create a new empty repo and give you access to it.

**Comment by visegura** [comment]
I agree with Tim's propositions, but right now I do not have so much time...
So yes Uemit please open a new repo, give me access to it and I put on my to do list Tim's proposition (at least the first 2 to start). It would be good also to change the link from the url provided in the paper to the new repo once done.

**Comment by visegura** [comment]
Oh and I'm happy with the proposed name

**Comment by timeu** [comment]
Ok I added a new repo (https://github.com/Gregor-Mendel-Institute/MultLocMixMod) pushed the current tip of this branch to the new repo. 

We should probably remove everything in this repo except a README.md which states that the repo was moved to the new location

**Comment by timflutre** [comment]
Thanks @timeu 

I started adding an issue on the new repo.

**Comment by timflutre** [comment]
@timeu @visegura maybe it's time to empty this repo, only leaving a `REAMDE.md` saying "the code moved to [MultLocMixMod](https://github.com/Gregor-Mendel-Institute/MultLocMixMod)"?

**Comment by Gill85** [comment]
Hi, I am new to the GWAS data analysis world. So could you please provide the clean Rscript of MLMM. Thanks!
